src/runtime/map_noswiss.go

```diff
@@ -358,7 +358,7 @@ func makemap(t *maptype, hint int, h *hmap) *hmap {
 // 1<<b is the minimum number of buckets to allocate.
 // dirtyalloc should either be nil or a bucket array previously
 // allocated by makeBucketArray with the same t and b parameters.
-// If dirtyalloc is nil a new backing array will be alloced and
```

Comment:
or is `alloced` to be read as: "alloc-ed", to have used `(m)alloc` on something? Still, `alloc` does indeed allocate memory, so `allocated` is correct, no?